buildpipeline/Core-Setup-PortableLinux-x64-master.json

```diff
@@ -0,0 +1,154 @@
+{
+  "build": [
+    {
```

Comment:
@wtgodbe I've added the Docker cleanup step.  Using `docker rm $(DockerContainerName)` isn't sufficient because it would fail if the container doesn't exist.  I was able to work around this by instead doing `docker ps -a -q --filter name=$(DockerContainerName) | xargs --no-run-if-empty docker rm -f`.  I had to use an inline script in the Shell++ build task for this, because the basic Run Command build task doesn't appear to support piping the `docker` command into `xargs`.